src/main.cpp

```diff
@@ -569,8 +573,63 @@ int main(int argc, char *argv[])
         }
     }
 
+    if (c == 'h') 
+    {
+         printf("Usage: wf-recorder [OPTION]... [FILE]...
+Screen recording of wlroots-based compositors
+
+With no FILE, start recording the current screen.
+
+  -a, --audio               Starts recording the screen with audio
+                
+  -c, --codec               Specifies the codec of the video. Supports  GIF output also.
+                            To modify codec parameters, use -p <option_name>=<option_value>
+                
+  -d, --device              Selects the device to use when encoding the video
+                            Some drivers report support for rgb0 data for vaapi input but really only support yuv.
+                            Use the -t or --to-yuv option in addition to the vaapi options to convert the data in software,
+                            before sending it to the gpu.
+                
+  -g, --geometry            Selects a specific part of the screen.
+  
+  -h, --help                Prints this help screen.
+                
+  -l, --log                 Generates a log on the current terminal. Debug purposes. 
+                
+  -o, --output              Specify the output where the video is to be recorded.
+                
+  -p, --codec-param         Change the codec parameters.
+                            -p <option_name>=<option_value>
+
+  -t, to-yuv                Use the -t or --to-yuv option in addition to the vaapi options to convert the data in software,
+                            before sending it to the gpu.
+                
+                
+                
+Examples:
+  
+  Video Only:
+  
+  - wf-recorder                       Records the video. Use Ctrl+C to stop recording. 
+                                      The video file will be stored as recording.mp4 in the current working directory.
+                
+  - wf-recorder -f <filename>         Records the video. Use Ctrl+C to stop recording. 
```

Comment:
The output name can be another one. For example, the user can also set `wf-recorder -f test.mkv` and then the resulting file will be `test.mkv` not `test.mp4`

Reply:
On the first example no arguments where given. And for what I have experienced it always results in the test.mp4. Probably I can add the information of the inexistence of  arguments

Reply:
It is just that the default file name is `recording.mp4`. If you specify a `-f test.mkv` then it should produce `test.mkv`. If it doesn't, then this is a bug.

Reply:
Let me clarify, I am giving 2 different examples on this print.

The first with the default value of the command `wf-recorder` without any arguments.
On the  the second, with a specific filename by running the command `wf-recorder -f <filename>` which will result in a recording with the name `filename`

Reply:
Perhaps you could elaborate the comment for the -f case (or both comments if necessary) to explain that `-f file.ext` means that the output file will be called `file.ext` and the file format will be guessed by the file extension `.ext`, which will fail if it's not one listed in the output of `ffmpeg -muxers`.

Reply:
The whole point is that the output file is not necessarily a .mp4 file, as the current help message says(and as far as I understand it)

Reply:
@ammen99  Sorry I only saw the .mp4 on the second example now, as I was answering on my phone :/ .  You were right  from the beginning. 
@soreau  I will try to address that.